app/src/main/res/menu/logged_in_menu.xml

```diff
@@ -3,16 +3,16 @@
     <item
         android:id="@+id/home"
         android:icon="@drawable/ic_home"
-        android:title="@string/TAB_TITLE_HOME" />
+        android:title="@string/home_tab.title" />
 
     <item
         android:id="@+id/insurance"
-        android:icon="@drawable/ic_claims"
```

Comment:
Make sure to remove this icon if it is no longer used

Reply:
This one is still used